compliance_checker/acdd.py

```diff
@@ -526,15 +526,12 @@ def verify_convention_version(self, ds):
                 if convention == 'ACDD-' + self._cc_spec_version:
                     return ratable_result((2, 2), None, []) # name=None so grouped with Globals
 
-                else:
-                    # Conventions attribute is present, but does not include
-                    # proper ACDD version
-                    messages = [
-                    "\"Conventions\" does not contain 'ACDD-{}'".format(self._cc_spec_version)
-                    ]
-                    return ratable_result((1, 2), None, messages)
+            # if no/wrong ACDD convention, return appropriate result
+            return ratable_result((0, 2), None, messages)
```

Comment:
Since it would have had a "Conventions" attribute, just not the right one, it should get a `(1, 2)` score